### PR TITLE
Support organization labels for PRs in API

### PR DESCRIPTION
**Pull request by CirnoT**
This PR fixes `/repos/{owner}/{repo}/pulls` and `/repos/{owner}/{repo}/pulls/{index}` to accept organization labels during PR creation and edition.

### CODE REVIEW

**Review by mrsdizzie — APPROVED** [review]

**Comment by mrsdizzie** [comment]
sorry I forgot these

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
I think this is a skip changelog as the feature was introduced in 1.12 and it's a bug fix for the feature.

File: routers/api/v1/repo/pull.go [hunk 1]
```diff
@@ -456,6 +468,18 @@ func EditPullRequest(ctx *context.APIContext, form api.EditPullRequestOption) {
 			ctx.Error(http.StatusInternalServerError, "ReplaceLabelsError", err)
 			return
 		}
+
+		if ctx.Repo.Owner.IsOrganization() {
+			labels, err = models.GetLabelsInOrgByIDs(ctx.Repo.Owner.ID, form.Labels)
```
Comment (6543) [inline routers/api/v1/repo/pull.go]: To Check: does this overwrite repo lables?

Comment (CirnoT) [reply routers/api/v1/repo/pull.go]: Good catch, I believe it will.

**Review by lafriks — DISMISSED** [review]
Block as per @6543 comment

File: routers/api/v1/repo/pull.go (hunk 1 above)
Comment (6543) [reply routers/api/v1/repo/pull.go]: I was not sure thats why I didnt approved/disaproved - had to look at the code ...
It did

File: routers/api/v1/repo/pull.go [hunk 2]
```diff
@@ -241,10 +241,22 @@ func CreatePullRequest(ctx *context.APIContext, form api.CreatePullRequestOption
...
 		}
 
-		labelIDs = make([]int64, len(labels))
+		labelIDs = make([]int64, len(form.Labels))
 		for i := range labels {
 			labelIDs[i] = labels[i].ID
 		}
+
+		if ctx.Repo.Owner.IsOrganization() {
+			labels, err = models.GetLabelsInOrgByIDs(ctx.Repo.Owner.ID, form.Labels)
```
Comment (6543) [inline routers/api/v1/repo/pull.go]: pleace make same fix here!

**Review by 6543 — CHANGES_REQUESTED** [review]

File: routers/api/v1/repo/pull.go (hunk 2 above)
Comment (CirnoT) [reply routers/api/v1/repo/pull.go]: Done

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
@lafriks I'm ok with it now
@CirnoT thanks for fixing 👍

**Comment by zeripath** [comment]
aw crap sorry I missed that error!

**Pull request merged**